style/style.css

```diff
@@ -0,0 +1,86 @@
+* {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  -webkit-box-sizing: border-box;
+          box-sizing: border-box; }
+
+body {
+  font-family: sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  background: #141d26; }
+
+.keyboard {
+  text-align: center;
+  margin: 0 auto;
+  width: 800px;
```

Comment:
I see that both keyboard and output elements have the same width value, so they can be placed in one container, which will have this 800px width and it's childs(keyboard, output) will inherit such width via width: 100%